ffi/capi/src/decimal.rs

```diff
@@ -92,14 +95,15 @@ pub mod ffi {
                 *new = char::from_u32(old).unwrap_or(char::REPLACEMENT_CHARACTER);
             }
             let digits = new_digits;
-            let plus_sign_affixes = AffixesV1 {
-                prefix: str_to_cow(plus_sign_prefix),
-                suffix: str_to_cow(plus_sign_suffix),
-            };
-            let minus_sign_affixes = AffixesV1 {
-                prefix: str_to_cow(minus_sign_prefix),
-                suffix: str_to_cow(minus_sign_suffix),
+            let strings = DecimalSymbolsV1Strings {
```

Comment:
Thought: should we name it a "builder"?

Check what I do with PackedPattern and potentially other places that do similar things